Formula/s/s3ql.rb

```diff
@@ -3,29 +3,27 @@ class S3ql < Formula
 
   desc "POSIX-compliant FUSE filesystem using object store as block storage"
   homepage "https://github.com/s3ql/s3ql"
-  url "https://github.com/s3ql/s3ql/releases/download/release-3.8.1/s3ql-3.8.1.tar.gz"
-  sha256 "d4731ebaacadca38a677bb18a99446c19d4f5b573628d55371f715acace11c4c"
-  license "GPL-3.0"
-  revision 1
+  url "https://github.com/s3ql/s3ql/releases/download/s3ql-5.1.2/s3ql-5.1.2.tar.gz"
+  sha256 "5921f6286247792ecb93cd78105eaa5d71e8b3037628ae1bce5c78dc6cf08fd2"
+  license "GPL-3.0-or-later"
```

Comment:
I think upstream used a deprecated license.

Reply:
Source doesn't include the "or any later version" in the file headers. It should be this:
```suggestion
  license "GPL-3.0-only"
```

Reply:
Ok right, it's fixed now